.github/workflows/build.yaml

```diff
@@ -0,0 +1,23 @@
+name: Build and Test
+
+on: [push, pull_request]
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+          python-version: ['2.7', '3.5', 'pypy2.7', 'pypy3.5']
+    steps:
+    - uses: actions/checkout@v3
+    - uses: actions/setup-python@v4
+      with:
+        python-version: ${{ matrix.python-version }}
+        architecture: x64
```

Comment:
this only affects windows, since there's no windows here this can be dropped

Reply:
Ah, I didn't know that. Thanks for the tip.

Reply:
https://docs.github.com/en/actions/automating-builds-and-tests/building-and-testing-python recommends using `actions/setup-python` to ensure consistency across runners. We also need to choose an explicit python version instead of the default.

Reply:
yeah I mean the `architecture` part specifically.  github's little window annoyingly expanded to more context lines